front/lib/api/assistant/actions/tables_query.ts

```diff
@@ -593,7 +593,8 @@ export class TablesQueryConfigurationServerRunner extends BaseActionConfiguratio
 
     if (
       "results" in sanitizedOutput &&
-      Array.isArray(sanitizedOutput.results)
+      Array.isArray(sanitizedOutput.results) &&
```

Comment:
shouldn't we test `sanitizedOutput.results` for nullity ?

Reply:
We should be covered with Array.isArray(null) and Array.isArray(undefined) that are both false, but thanks for making me check that indeed 🙇🏻‍♀️